pymatgen/entries/compatibility.py

```diff
@@ -437,10 +632,21 @@ def process_entry(self, entry):
             returned.
         """
         try:
-            corrections = self.get_corrections_dict(entry)
-        except CompatibilityError:
+            corrections, errors = self.get_corrections_dict(entry)
+        except CompatibilityError as error:
+            print("CompatibilityError: " + str(error))
             return None
         entry.correction = sum(corrections.values())
+        tot_error = 0
+        for e in errors.values():
+            tot_error += e ** 2
+        tot_error = sqrt(tot_error)
+        if tot_error == 0:
+            # if there are no error values available for the corrections applied,
+            # set correction_uncertainty to not a number
+            entry.data["correction_uncertainty"] = float("NaN")
```

Comment:
@awvio can you change `float("NaN")` to `np.nan`? The float('NaN') code causes problems with `as_dict` / `from_dict`.